test_conformance/images/kernel_image_methods/test_1D_buffer.cpp

```diff
@@ -67,14 +67,13 @@ static int test_get_1Dimage_buffer_info_single(cl_context context,
     if (gDebugTrace)
         log_info(" - Creating 1D image %d ...\n", (int)imageInfo->width);
 
-    buffer = clCreateBuffer(
-        context, flags, imageInfo->width * get_pixel_size(imageInfo->format),
-        NULL, &error);
+    buffer =
+        clCreateBuffer(context, flags, imageValues.getSize(), NULL, &error);
```

Comment:
Note, there is a merge conflict here now, caused by the recently merged https://github.com/KhronosGroup/OpenCL-CTS/pull/2073.  It's a little different than the change in this PR, though.  Is there a reason to prefer one fix vs. the other?

Reply:
No, AFAICT they are both the exact same number (row pitch). I'll drop the patch